xml/operations-configuring-identity-websso.xml

```diff
@@ -52,8 +52,17 @@
    </listitem>
    <listitem>
     <para>
-     Only WebSSO federation using the SAML method is supported in
-     &product; . OpenID-based federation is not currently supported.
+     The &productname; WebSSO function with OpenID method was tested with
+     Google OAuth 2.0 APIs, which conforms to the OpenID Connect specification.
```

Comment:
Either say "with the Google OAuth 2.0 APIs, which conform" if it really is more than one API, or "with the Google OAuth 2.0 API, which conforms" if it is just one API.

Reply:
Done